app/views/staff/activities/financials.html.haml

```diff
@@ -35,14 +35,15 @@
             = t("page_title.activity.financials")
 
           - if @activity.fund? && policy(:fund).create?
+            = render partial: "staff/shared/activities/budgets", locals: { activity: @activity, budget_presenters: @budget_presenters }
```

Comment:
Took me a while to understand what had changed, because of the way GitHub has displayed the diff here!